src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -71,4 +71,10 @@ private Line findLineById(Long id) {
 		return lineRepository.findById(id)
 			.orElseThrow(() -> new IllegalArgumentException("노선이 존재하지 않습니다."));
 	}
+
+	public void deleteStationByIdInLine(Long lineId, Long stationId) {
```

Comment:
이 메서드가 수행되는 객체의 이름이 라인 서비스인데 메서드명에 또 표시할 필요가 있을까요
그리고 변수명으로 의미를 전달 할 수 있으니 이를 활용해보아요

```suggestion
	public void deleteStation(Long lineId, Long stationId) {
```